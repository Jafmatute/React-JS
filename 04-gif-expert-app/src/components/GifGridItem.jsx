export const GifGridItem = ({title, url}) => (

<div className="card">
<img src={ url } alt={title} />
</div>

);
