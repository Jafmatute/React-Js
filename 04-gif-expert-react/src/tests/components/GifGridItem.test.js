import { shallow } from 'enzyme';

import GifGridItem from '../../components/GifGridItem';

describe('Pruebas al componente <GifGridItem />', () => {
  test('Debe renderizar el componente correctamente <GifGridItem /', () => {
    const title = 'Buscando empleo';
    const url = 'https://google.com';

    const wrapper = shallow(<GifGridItem />);

    expect(wrapper).toMatchSnapshot();
  });
});
