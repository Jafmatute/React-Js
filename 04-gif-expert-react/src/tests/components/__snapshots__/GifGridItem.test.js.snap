// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas al componente <GifGridItem /> Debe renderizar el componente correctamente <GifGridItem / 1`] = `
<div
  className="card animate__animated animate__fadeIn"
>
  <img />
  <p />
</div>
`;
